oddbird/oddleventy
# quotes

![photo-1504618223053-559bdef9dd5a](https://user-images.githubusercontent.com/8933115/182471145-ca43f88f-2f6e-4b94-a077-79c33f5d54f7.jpeg)






## Steps to test/reproduce
- See new quotes on Work list page
- See new quotes on Metecho and MetaDeploy detail pages
- Second paragraph at the top of /services/development/ (removed word "modern")
